doc/manual/installation.md

```diff
@@ -0,0 +1,341 @@
+
+    sudo -u huginn -H bundle exec rake db:seed RAILS_ENV=production SEED_USERNAME=admin SEED_PASSWORD=yourpassword
+
+### Compile Assets
+
+    sudo -u huginn -H bundle exec rake assets:precompile RAILS_ENV=production
+
+### Install Init Script
+
+Huginn uses foreman to generate the init scripts based on a `Procfile`
+
+Edit the `Procfile` and choose one of the suggested versions for production
+
+    sudo -u huginn -H editor Procfile
+
+Export the init scripts:
+
+    sudo rake production:export
+
+**Note:** You have to re-export the init script every time you change the configuration in `.env` or your `Procfile`!
+
+### Setup Logrotate
+
+    sudo cp lib/support/logrotate/huginn /etc/logrotate.d/huginn
+
+
+### Ensure Your Huginn Instance Is Running
+
+    sudo rake production:status
+
+## 7. Nginx
+
+**Note:** Nginx is the officially supported web server for Huginn. If you cannot or do not want to use Nginx as your web server, the wiki has a page on how to configure [apache](https://github.com/cantino/huginn/wiki/Apache-Huginn-configuration).
+
+### Installation
+
+    sudo apt-get install -y nginx
+
+### Site Configuration
+
+Copy the example site config:
+
+    sudo cp lib/support/nginx/huginn /etc/nginx/sites-available/huginn
+    sudo ln -s /etc/nginx/sites-available/huginn /etc/nginx/sites-enabled/huginn
+
+Make sure to edit the config file to match your setup, if you are running multiple nginx sites remove the `default_server` argument from the `listen` directives:
+
+    # Change YOUR_SERVER_FQDN to the fully-qualified
+    # domain name of your host serving Huginn.
+    sudo editor /etc/nginx/sites-available/huginn
+
+Remove the default nginx site, **if huginn is the only enabled nginx site**:
+
+    sudo rm /etc/nginx/sites-enabled/default
+
+**Note:** If you want to use HTTPS, replace the `huginn` Nginx config with `huginn-ssl`. See [Using HTTPS](#using-https) for HTTPS configuration details.
+
+### Test Configuration
+
+Validate your `huginn` or `huginn-ssl` Nginx config file with the following command:
+
+    sudo nginx -t
+
+You should receive `syntax is okay` and `test is successful` messages. If you receive errors check your `huginn` or `huginn-ssl` Nginx config file for typos, etc. as indicated in the error message given.
+
+### Restart
+
+    sudo service nginx restart
+
+# Done!
+
+### Initial Login
+
+Visit YOUR_SERVER in your web browser for your first Huginn login. The setup has created a default admin account for you. You can use it to log in:
+
+    admin
+    password
+
+
+**Enjoy!**
+
+You can use `sudo start huginn` and `sudo stop huginn` to start and stop Huginn.
```

Comment:
Maybe another line below:

Be sure to read the section about how to update your Huginn installation as well!